l10n_br_hr/model/hr_employee.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# (c) 2014 Kmee - Rafael da Silva Lima <rafael.lima@kmee.com.br>
+# (c) 2014 Kmee - Matheus Felix <matheus.felix@kmee.com.br>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields, api
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+from openerp.exceptions import Warning as UserError
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.constrains('pis_pasep')
+    def _validate_pis_pasep(self):
```

Comment:
Maybe this function should be in fiscal.py in l10n_br_base/tools/ to be reusable.